clients/elrond/client.go

```diff
@@ -135,9 +138,13 @@ func checkArgs(args ClientArgs) error {
 	if check.IfNil(args.TokensMapper) {
 		return clients.ErrNilTokensMapper
 	}
-	if args.MaxRetriesOnQuorumReached < minRetriesOnQuorum {
+	if args.MaxRetriesOnQuorumReached < minRetries {
```

Comment:
since these are tight to the bridge struct operation, we might think of moving them in the bridge struct. 
Not to be done in this PR.

Reply:
done in #206